site/en/blog/lighthouse-9-0/index.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: 'layouts/blog-post.njk'
+title: What's new in Lighthouse 9.0
+authors:
+  - brendankenny
+date: 2021-11-12
+description: >
+  Lighthouse 9.0 arrives with a refreshed report and a preview of support for user flows.
+hero: 'image/MtjnObpuceYe3ijODN3a79WrxLU2/hcf9uCrSxA4VNMiXHsVn.png'
+alt: 'A new Lighthouse 9.0 report'
+tags:
+  - new-in-lighthouse
+  - lighthouse
+  - chrome-98
+---
+
+Lighthouse is a website auditing tool that helps developers with opportunities and diagnostics to improve the user experience of their sites.
+
+Lighthouse 9.0 is available immediately on the command line and in [Chrome Canary](https://www.google.com/chrome/canary/).
+It will land in Chrome stable in Chrome 98 and be available in [PageSpeed Insights](https://developers.google.com/speed/pagespeed/insights/) within a week.
+
+## API changes
+
+Most users should not run into any workflow-breaking changes with this release. If you run custom Lighthouse audits or use tools that depend on details deep in the Lighthouse report JSON, there may be some breaking changes in 9.0 that you need to be aware of.
+
+See the full list of changes in the [9.0 changelog](https://github.com/GoogleChrome/lighthouse/releases/tag/v9.0.0).
+
+## Report refresh {: #report-refresh }
+
+The Lighthouse report has been refreshed to improve readability and make the source of the report and how it was run clearer.
+
+A final screenshot has been embedded at the top of the report to make it obvious at a glance if the page being tested loaded correctly and is in the format expected.
+
+{% Img src="image/MtjnObpuceYe3ijODN3a79WrxLU2/VTimm2GLjvFXwywBi9mn.png", alt="A Lighthouse 9.0 report", width="800", height="629", class="screenshot" %}
```

Comment:
A tough one to do for some of these, but I would go into some detail in the alt text. Doesn't have to be excessive, but maybe point out some of the high level details of the screenshot to help paint a better mental picture.

Reply:
> A tough one to do for some of these, but I would go into some detail in the alt text.

thanks for flagging this. I think I've improved all these but happy to iterate again.